pkg/ovssubnet/controller/multitenant/bin/openshift-sdn-multitenant-setup.sh

```diff
@@ -176,9 +176,12 @@ function setup() {
     sysctl -w net.ipv4.ip_forward=1
     sysctl -w net.ipv4.conf.${TUN}.forwarding=1
 
-    # delete the subnet routing entry created because of lbr0
+    # delete the subnet routing entry created because of ip addr on lbr0
     ip route del ${local_subnet_cidr} dev lbr0 proto kernel scope link src ${local_subnet_gateway} || true
 
+    # delete the subnet routing entry created because of ip addr on ${TUN}
+    ip route del ${local_subnet_cidr} dev ${TUN} proto kernel scope link src ${local_subnet_gateway} || true
```

Comment:
i think it would make more sense to do this immediately after adding the ${cluster_network_cidr} route on tun0

Also, kube plugin?